### DISCUSSION TITLE
[Issue?] Dynamic Prompt seems a little 'noisy' in the console

### DISCUSSION THREAD

**Discussion by brknsoul** [adieyal/sd-dynamic-prompts#489] [General]
After a recent-ish update, Dynamic Prompt is a little verbose than it used to be;

Note that I don't have any characters in my prompt that would trigger Dynamic Prompting (eg: `{ | } __` ).

![image](https://github.com/adieyal/sd-dynamic-prompts/assets/1969381/485ffa13-eb85-4287-949b-30f7f05b9a0d)

**Comment by adieyal** [comment]
try update to the latest version, that should remove some of the logging
